app/components/wallet/skins/AmountInputSkin.js

```diff
@@ -40,12 +39,33 @@ export default class AmountInputSkin extends Component<Props> {
   };
 
   render() {
-    const { error, fees, total, currency, classicTheme } = this.props;
+    const {
+      currency,
+      fees,
+      total,
+      error,
+      classicTheme,
+      inputRef,
+      theme,
+      themeId,
+      value,
+      type,
+    } = this.props;
     const { intl } = this.context;
 
     return (
       <div className={styles.component}>
-        {classicTheme ? <InputSkin {...this.props} /> : <InputOwnSkin {...this.props} />}
```

Comment:
This is one of those cases where I mentioned we don't use exact props on purpose in order to be able to pass arbitrary props down to child components (you can think of it like inheritance).
The only way we could make this exact is by modifying the `React-Polymorph` library to make it export its Prop types for the skins and then do a type-level union (which is maybe not a bad idea to make a PR for this to React-Polymorph)

Reply:
I was aware of the situation but I thought the above reason only applied to the `<InputSkin {...this.props} />` part. In the case of `<InputOwnSkin>` (https://github.com/Emurgo/yoroi-frontend/blob/develop/app/themes/skins/InputOwnSkin.js#L65) the extra properties in AmountInputSkin.js are not actually passed down anywhere. However this change seems a little bit aggressive to me indeed... Should I revert?

Reply:
Ah I was wrong. Scratch above comment.

Reply:
Yeah, props types of skins should be not be exact given the way react-polymorph is.